### PR TITLE
fix: update renames in codelabs and examples to v7

### LINKED ISSUE #884: Update goog_module renames in codelabs
**Category**

 - Codelabs

**Describe the bug**

Codelabs still have the old names for things that were renamed..

Eg:
  * getMainWorkspace: [here](https://github.com/google/blockly-samples/blob/1be2b748414ef147c8f7d657fa440442c0ef49a8/examples/getting-started-codelab/complete-code/scripts/main.js#L12), [here](https://github.com/google/blockly-samples/blob/1be2b748414ef147c8f7d657fa440442c0ef49a8/codelabs/getting_started/getting_started.md), [here](https://github.com/google/blockly-samples/blob/1be2b748414ef147c8f7d657fa440442c0ef49a8/codelabs/custom_toolbox/custom_toolbox.md)

### PR DESCRIPTION
**Pull request by BeksOmega**
### Description

Uses the new renamings script to automatically perform renames in codelabs and samples.

Closes #884 , but this will be a continual problem haha.

### CODE REVIEW

File: examples/pitch-field-demo/field_pitch.js [hunk 1]
```diff
@@ -91,10 +91,10 @@ CustomFields.FieldPitch.prototype.showEditor_ = function() {
   // change this behaviour.  For now, using bindEvent_ instead of
   // bindEventWithChecks_ allows it to work without a mousedown/touchstart.
   this.clickWrapper_ =
-      Blockly.bindEvent_(this.imageElement_, 'click', this,
```
Comment (BeksOmega) [inline examples/pitch-field-demo/field_pitch.js:94]: We actually left aliases for these, so the renamings database needs to be modified.

File: codelabs/custom_generator/custom_generator.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ The blocks are:
 Copy this code into `custom_generator.js` to define the two custom blocks.
 
 ```js
-Blockly.defineBlocksWithJsonArray([{
```
Comment (BeksOmega) [inline codelabs/custom_generator/custom_generator.md:71]: Turns out this also need aliases in the file. See https://github.com/google/blockly/pull/5699

File: examples/custom-dialogs-demo/custom-dialog.js [hunk 3]
```diff
@@ -13,16 +13,16 @@
  */
 CustomDialog = {};
 
-/** Override Blockly.alert() with custom implementation. */
-Blockly.alert = function(message, callback) {
```
Comment (BeksOmega) [inline examples/custom-dialogs-demo/custom-dialog.js:17]: These need backwards compatible paths as well.

File: codelabs/custom_toolbox/custom_toolbox.md [hunk 4]
```diff
@@ -114,7 +114,7 @@ However, if you run the below commands in your console you will see that
 your toolbox is now using the `CustomCategory` class.
 
 ```js
-var toolbox = Blockly.getMainWorkspace().getToolbox();
```
Comment (BeksOmega) [inline codelabs/custom_toolbox/custom_toolbox.md:117]: Also `mainWorkspace` and `getMainWorkspace`

File: examples/pitch-field-demo/field_pitch.js [hunk 5]
```diff
@@ -34,14 +34,14 @@ CustomFields.FieldPitch = function(text) {
 
   /**
    * Click event data.
-   * @type {?Blockly.EventData}
```
Comment (BeksOmega) [inline examples/pitch-field-demo/field_pitch.js:37]: afaict we don't actually export EventData anymore. Leads to the question: should we?

**Comment by BeksOmega** [comment]
I misunderstood what `newPath` meant. We only added `newPath`s to things that were *only* exported at their old place in the Blockly tree. Not things that were *also* exported at their old path. So all of the migrations were correct.

File: examples/pitch-field-demo/field_pitch.js [hunk 6]
```diff
@@ -34,14 +34,14 @@ CustomFields.FieldPitch = function(text) {
...
+   * @type {?Blockly.eventHandling.Data}
    * @private
    */
   this.clickWrapper_ = null;
 
   /**
    * Move event data.
-   * @type {?Blockly.EventData}
+   * @type {?Blockly.eventHandling.Data}
```
Comment (alschmiedt) [inline examples/pitch-field-demo/field_pitch.js]: Should this be `Blockly.browserEvents.Data`? I can't seem to find `Blockly.eventHandling` in core.

**Review by alschmiedt — APPROVED** [review]

**Pull request merged**